spec/rails_helper.rb

```diff
@@ -1,4 +1,5 @@
 ENV['RAILS_ENV'] ||= 'test'
+require 'simplecov'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.